src/components/Inquiry/Answer.jsx

```diff
@@ -5,13 +5,23 @@ import store from '../../state/store/configureStore'
 import { useTranslation } from 'react-i18next'
 import CustomRadioButton from '../../components/custom/CustomRadioButton'
 
-const Answer = ({ text, type, placeholder, questionKey }) => {
+const Answer = ({
+  text,
+  type,
+  placeholder,
+  questionKey,
+  label_1,
+  label_2,
+  dataCy_1,
+  dataCy_2,
+  value_1,
+  value_2,
```

Comment:
Could these be stored in arrays / objects, just to to get rid of a bit of em?

Reply:
refactered